### PR TITLE
Steam Test Server - Update 7

### PR DESCRIPTION
**Pull request by Sunaries**
https://github.com/Treeofsavior/EnglishTranslation/commit/328f57b2f4602330aa1af8dc7315cb11262511d1 Proofread and fixed errors. + Fixed a few inconsistencies and re-named few NPCs and titles.

GitHub Logo
The Token item description was not up-to-date. It is now.

@imcgames 
I have also encountered a bug after the recent Steam update, you will need to see the **#bug-report** channel about it.

Q1 - I'd like to evaluate the Guild UI and the UI of other contents. But I do not have the tools for it. 
Wanna help me out?

Q2 - You've removed the {memo x} tags from all the files. This means that old, outdated and mistranslated text remains. Are you using any of these, for Skills or other contents in the game?

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -12000,85 +12000,85 @@ QUEST_LV_0100_20151224_011996
...
-QUEST_LV_0100_20151224_012041	I don't know how many times you've protected our townsfolk on the way here..{nl}Yet we receive your help again. Thank you so much.{nl}
+QUEST_LV_0100_20151224_012040	How many migration orders did Orsha issue anyways...{nl}I don't know how much longer we will have to just wait here.
+QUEST_LV_0100_20151224_012041	I don't know how many times you've protected our townsfolk on the way here...{nl}Yet we receive your help again. Thank you so much.{nl}
 QUEST_LV_0100_20151224_012042	By the way, how are the nightmares now?{nl}How about consulting the bishop in Orsha when you get there?
-QUEST_LV_0100_20151224_012043	The lord of Orsha should have issued Migration Orders sooner.{nl}There were too many casualties, too many..{nl}
-QUEST_LV_0100_20151224_012044	It's in times like this when you need something to rely upon..{nl}Where have the goddesses gone to..
-QUEST_LV_0100_20151224_012045	They said that the lord will send soldiers and officials to help if we managed to get here..{nl}I don't see any sign of them.{nl}
+QUEST_LV_0100_20151224_012043	The lord of Orsha should has issued the migration orders sooner.{nl}There were too many casualties. Too many...{nl}
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:12046]: should has? I think the "should have" is correct. 

I would also change the last expression to "Way too many". (just my preference though :P)

File: QUEST.tsv [hunk 2]
```diff
@@ -3491,48 +3491,48 @@ QUEST_20151224_003498	{nl}The small boat was unable to withstand the combined we
...
 QUEST_20151224_003525	There seems to be something nearby.
 QUEST_20151224_003526	The grave guard's spell doll attacks the sacred object of the religious body.
 QUEST_20151224_003527	Gimmick Scarecrow
 QUEST_20151224_003528	Gimmick Statue
 QUEST_20151224_003529	Everything is becoming clearer now.{nl}I thank you once more.
-QUEST_20151224_003530	My hometown of Delmore Castle.. Is a truly beautiful place.{nl}The neighbors are so kind and the lord is also generous.{nl}
-QUEST_20151224_003531	I was able to receive many rewards and apply for a long vacation thanks to my diligent service as a liason officer.{nl}I was excited at the thought of hosting a feast with the neighbors.{nl}
-QUEST_20151224_003532	But what I saw when I arrived home.. {nl}Was an empty town with nobody in sight.{nl}
-QUEST_20151224_003533	Well, to be completely fair.. It wasn't totally empty. {nl}Demons suddenly appeared from every direction.{nl}
+QUEST_20151224_003530	My hometown of the Delmore Castle... is a truly beautiful place.{nl}The neighbors are so kind and the lord is also generous.{nl}
```
Comment (ttgmichael) [inline QUEST.tsv]: Strange way of wording this in English.

My hometown, the Delmore Castle... is a truly beautiful place. Is less wordy and gets the point across imo.

File: QUEST.tsv [hunk 3]
```diff
@@ -3491,48 +3491,48 @@ QUEST_20151224_003498	{nl}The small boat was unable to withstand the combined we
...
+QUEST_20151224_003531	I was able to receive many rewards and apply for a long vacation {nl}thanks to my diligent service as a liason officer.{nl}I was excited at the thought of hosting a feast with the neighbors.{nl}
+QUEST_20151224_003532	But what I saw when I arrived home... {nl}It was an empty town with nobody in sight.{nl}
```
Comment (ttgmichael) [inline QUEST.tsv]: I think the person is trying to finish their sentence from the line before. Remove the 'it' and keep the 'was' lowercase?

File: QUEST.tsv [hunk 4]
```diff
@@ -3491,48 +3491,48 @@ QUEST_20151224_003498	{nl}The small boat was unable to withstand the combined we
...
+QUEST_20151224_003533	Well, to be completely fair... It wasn't totally empty. {nl}Demons suddenly appeared from every direction.{nl}
```
Comment (ttgmichael) [inline QUEST.tsv]: to be completely truthful/honest?

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -12000,85 +12000,85 @@ QUEST_LV_0100_20151224_011996
...
+QUEST_LV_0100_20151224_012044	It's in times like this when you need something to rely upon...{nl}Where have the goddesses gone to...
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:12047]: It is in times like this when you need someone to have faith in...? ('have faith in' is to 'rely' on something thru faith, imo)

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -12000,85 +12000,85 @@ QUEST_LV_0100_20151224_011996
...
+QUEST_LV_0100_20151224_012045	They said that the lord will send soldiers and officials to help if we managed to get here...{nl}I don't see any sign of them.{nl}
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:12048]: They said that the lord will send soldiers and officials to help

They said that the lord would send soldiers and officials to help

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -12000,85 +12000,85 @@ QUEST_LV_0100_20151224_011996
...
-QUEST_LV_0100_20151224_012052	Officer Lutas is saying that there are too many refugees to let them all into Orsha.{nl}That's why they've made and are offering aid to several smaller refugee campes near Orsha.{nl}
-QUEST_LV_0100_20151224_012053	Aid.. I don't know if you can call this proper aid.
-QUEST_LV_0100_20151224_012054	4 years have passed since Medzio Diena, but the reconstruction of Orsha is still underway.{nl}There are some refugees that are disappointed when they see Orsha.. But it should still be better than other places.
-QUEST_LV_0100_20151224_012055	A person with your skills is welcome to enter Orsha at once.{nl}Go to the castle next to the central plaza if you wish to meet the lord.
-QUEST_LV_0100_20151224_012056	Thank you so much for saving me!{nl}Forget about Orsha, I almost.. Whew.. {nl}
-QUEST_LV_0100_20151224_012057	You helped us from the monsters on the way here as well.{nl}Even though the goddesses are nowhere to be seen, I can't help but think that they are still somewhere when I look at you.{nl}
+QUEST_LV_0100_20151224_012048	Eh? Don't you hear something strange?{nl}I think something is coming this way...
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:12051]: Hmm? Do you hear something strange?

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -13096,13 +13096,13 @@ QUEST_LV_0100_20151224_013092	Yane, stop being stubborn and take care of Melchio
 QUEST_LV_0100_20151224_013093	We're lagging behind.{nl}Let's hurry and head to the Odagingas Vacant Lot!{nl}
 QUEST_LV_0100_20151224_013094	It's over!{nl}
 QUEST_LV_0100_20151224_013095	What happened to Delmore Rephaim?{nl}What was all that light and the ground shaking...?{nl}
-QUEST_LV_0100_20151224_013096	Is that true...? So he had the end he deserved.{nl}I just don't know if this can appease all the Delmore Fortress {nl}residents that are now with the goddess.
+QUEST_LV_0100_20151224_013096	Is that true...? So he had the end he deserved.{nl}I just don't know if this can appease all the Delmore Castle {nl}residents that are now with the goddess.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:13099]: Is that so? So he got the end he deserved.

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -13096,13 +13096,13 @@ QUEST_LV_0100_20151224_013092	Yane, stop being stubborn and take care of Melchio
...
 QUEST_LV_0100_20151224_013097	Delmore Rephaim is gone but... I'm still worried about Melchioras.{nl}Shall we go back now?
 QUEST_LV_0100_20151224_013098	What... what happened to Delmore Rephaim?{nl}
 QUEST_LV_0100_20151224_013099	I see...{nl}I guess it went as expected from him then...{nl}
 QUEST_LV_0100_20151224_013100	Right before he escaped he seemed almost addicted to forbidden knowledge, you see.{nl}As if there was nothing that could stop him...{nl}
 QUEST_LV_0100_20151224_013101	And I followed him, knowing fully well he was an ally of the demons.{nl}But there was no match for his character, his thirst for knowledge.{nl}
-QUEST_LV_0100_20151224_013102	When I found out he was going to use the people of Delmore Fortress to make the Kruvina...{nl}Yes, had I been able to stop him then, everyone would have been saved...{nl}
+QUEST_LV_0100_20151224_013102	When I found out he was going to use the people of the Delmore Castle to make the Kruvina...{nl}Yes, had I been able to stop him then, everyone would have been saved...{nl}
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:13105]: When I found out he was going to use the people of the Delmore Castle to make the Kruvina...{nl}Well,

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -13113,7 +13113,7 @@ QUEST_LV_0100_20151224_013109	Melchioras has managed to regain his conscience.{n
 QUEST_LV_0100_20151224_013110	Melchioras regained his conscience while we were fighting, it seems.{nl}I'm glad everything is going to be alright now. I really am.
 QUEST_LV_0100_20151224_013111	Melchioras was asking for you the whole time.{nl}I don't even know what to say, I'm so glad everything ended well...
 QUEST_LV_0100_20151224_013112	I hoped I could be the one to catch Delmore Rephaim...{nl}But I'm glad it turned out well. If only I wasn't hurt...
-QUEST_LV_0100_20151224_013113	Chasing away the demons, rebuilding the walls...{nl}I wonder how much time it would take to make Delmore Fortress a place worth living again.
+QUEST_LV_0100_20151224_013113	Chasing away the demons, rebuilding the walls...{nl}I wonder how much time it would take to make the Delmore Castle a place worth living again.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:13116]: I wonder how long will it take to make the Delmore Castle a place worth living again.

File: QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -13131,7 +13131,7 @@ QUEST_LV_0100_20151224_013127	Use the crystal on the magic supply device to dest
 QUEST_LV_0100_20151224_013128	I'm really not trying to blame Yane but...{nl}I just want to make sure we do things right.
 QUEST_LV_0100_20151224_013129	Thank you... so much.{nl}It should be impossible to create another Kruvina now.{nl}
 QUEST_LV_0100_20151224_013130	I can't rid myself of my sins, nor can I take them back.{nl}All I can do is stay here and continue to repent.
-QUEST_LV_0100_20151224_013131	Nothing guarantees there won't another person like Delmore Rephaim in the future.{nl}Still, it's a pity what happened to Delmore Fortress...
+QUEST_LV_0100_20151224_013131	Nothing guarantees there won't another person like Delmore Rephaim in the future.{nl}Still, it's a pity what happened to the Delmore Castle...
 QUEST_LV_0100_20151224_013132	Thank you so much for rescuing Melchioras.{nl}I don't know what would be of this guilt if he hadn't made it...
 QUEST_LV_0100_20151224_013133	Thank you so much for rescuing Melchioras.{nl}I don't know what would be of this guilt if he hadn't made it...
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:13136]: I would not be able to handle the guilt if he hadn't made it

File: QUEST.tsv [hunk 12]
```diff
@@ -3345,7 +3345,7 @@ QUEST_20151224_003352	Paladins are fearless because we believe the goddesses wil
 QUEST_20151224_003353	Now it's turn to test the techniques you've learned so far.{nl}Go and defeat the monsters in Apsimesti Crossroads.
 QUEST_20151224_003354	They say everything works out if you just devote yourself, right?{nl}Orsha and Fedimian are a bit far away but... please think of it as devotion.
 QUEST_20151224_003355	Ardel's Diary Page 1
-QUEST_20151224_003356	Date: XX/XX. Clear sky.{nl}-{nl}Things at the Delmore Fortress are busy as always.{nl}The walls destroyed during Medzio Diena are being rebuilt and everyone is returning to their normal lives.{nl}I  hope nothing like Medzio Diena happens ever again.{nl}Thanks to Lord Delmore I think the fortress will be safe now.
+QUEST_20151224_003356	Date: XX/XX. Clear sky.{nl}-{nl}Things at the Delmore Castle are busy as always.{nl}The walls destroyed during Medzio Diena are being rebuilt and everyone is returning to their normal lives.{nl}I  hope nothing like Medzio Diena happens ever again.{nl}Thanks to Lord Delmore I think the fortress will be safe now.
```
Comment (ttgmichael) [inline QUEST.tsv]: There's two spaces between the I and hope in "I  hope nothing like Medzio Diena happens ever again."

**Comment by Sunaries** [comment]
Thanks @ttgmichael :+1: 

Good to merge.

**Comment by Sunaries** [comment]
Thanks @ttgmichael :+1: 

Good to merge.

**Comment by Sunaries** [comment]
Thanks @ttgmichael :+1: 

Good to merge.

**Comment by Sunaries** [comment]
Thanks @ttgmichael :+1: 

Good to merge.

**Comment by Sunaries** [comment]
Thanks @ttgmichael :+1: 

Good to merge.

**Comment by imcgames** [comment]
@Sunaries 

About "bug report", please check the discord.

Q1 - I'd like to evaluate the Guild UI and the UI of other contents. But I do not have the tools for it. 
Wanna help me out?
 -> Actually, there is no way to check the guild UI in directly. The Templar class only can see and use it. (Dev teams are also same condition.) So I think the best way is to provide you lots of EXP Cards for booster via the message box. I will let you know before I start.

Q2 - You've removed the {memo x} tags from all the files. This means that old, outdated and mistranslated text remains. Are you using any of these, for Skills or other contents in the game?
 -> Yes, we are using it for other contents in game. We will translate it according to priority. Just please proofreading. :+1:

**Pull request merged**